webpack.config.js

```diff
@@ -69,10 +69,10 @@ const createServerConfig = (_env, mode) => {
   });
 
   if (mode === "development") {
-    config.cache = {
-      type: "filesystem",
-      name: "server",
-    };
+    // config.cache = {
+    //   type: "filesystem",
+    //   name: "server",
+    // };
```

Comment:
It took a while to find these... doing `lemmy-js-client` work, using `yalc publish --push`, wasn't updating that dependency in lemmy-ui. Finally found that if I removed the entire node_modules folder, it would rebuild, then finally found that this was the culprit.

Reply:
Do you think it's worth removing completely instead of just commenting out?